CHANGELOG.md

```diff
@@ -8,8 +8,11 @@ All notable changes to this extension will be documented in this file.
 
 - If the route call fails while fetching Schema Registry subject names while populating the topics
   view, show a warning but continue on with no schema associations for the topics.
+- Sidecar version update to include a fix for listing subjects.
 - Selecting resources from a quickpick should no longer throw a "No current organization found"
   error when the user is not logged into Confluent Cloud.
+- Selecting the 'generate project from template' context item on a single direct connection topic
```

Comment:
Minor wording change for accuracy:
```suggestion
- Selecting the 'Generate Project from Resource' context item on a single direct connection topic
```

Reply:
pushed the change

Reply:
Looks like the `should no longer display an error.` part was dropped - intentional?

Reply:
no --- pushed fix